rpcs3/Emu/NP/signaling_handler.cpp

```diff
@@ -541,7 +542,11 @@ void signaling_handler::start_sig2(u64 room_id, u16 member_id)
 	auto& sent_packet   = sig2_packet;
 	sent_packet.command = signal_connect;
 
-	auto si = sig2_peers.at(room_id).at(member_id);
+	ensure(sig2_peers.contains(room_id));
+	const auto& sp = sig2_peers.at(room_id);
+
+	ensure(sp.contains(member_id));
+	const auto& si = sp.at(member_id);
```

Comment:
I'd rather not have reference to shared_ptr, that's not in the spirit of the container.

Reply:
made explicit copy of the shared pointers to reduce IDE warnings about ambiguity